src/EditorFeatures/VisualBasic/LineCommit/CommitBufferManager.vb

```diff
@@ -114,6 +114,19 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
                         Return
                     End If
 
+                    If Not isExplicitFormat Then
+                        ' Avoid automatically formatting extremely large dirty spans without an explicit request from
+                        ' the user. The "large span threshold" is 7000 lines.
```

Comment:
Please justify where 7000 came from, just so we know if/when to reconsider this.

Reply:
Amended commit to include better comment and also commit message links to the internal customer issue.